pyemma/coordinates/api.py

```diff
@@ -1168,10 +1199,95 @@ def tica(data=None, lag=10, dim=-1, var_cutoff=0.95, kinetic_map=True, commute_m
             user_msg,
             category=PyEMMA_DeprecationWarning)
 
-    res = TICA(lag, dim=dim, var_cutoff=var_cutoff, kinetic_map=kinetic_map, commute_map=commute_map, skip=skip)
+    res = TICA(lag, dim=dim, var_cutoff=var_cutoff, kinetic_map=kinetic_map, commute_map=commute_map, skip=skip,
+               weights=weights, reversible=reversible)
     return _param_stage(data, res, stride=stride)
 
 
+def lagged_covariance(data=None, c00=True, c0t=True, ctt=False, remove_constant_mean=None, remove_data_mean=False,
+                      reversible=False, bessel=True, lag=0, reweighting="empirical", stride=1, skip=0, chunksize=None):
+    """
+        Compute lagged covariances between time series. If data is available as an array of size (TxN), where T is the
+        number of time steps and N the number of dimensions, this function can compute lagged covariances like
+
+        .. math::
+
+            C_00 &=      X_t X \\
+            C_{0t} &= X_t Y \\
+            C_{tt} &= Y_t Y,
```

Comment:
On the RHS there should be no t index. Alternatively use X_0 and X_t everywhere.